### PR TITLE
Add error handling for showing plan error

### PR DESCRIPTION
**Pull request by jesusbv**
- Handle show plan made with different version
- Update specs to avoid that error
- Update specs cover so that case is considered

### CODE REVIEW

**Comment by bear454** [comment]
Please rebase to resolve the test suite issue.

File: spec/support/helpers.rb [hunk 1]
```diff
@@ -21,11 +21,16 @@ def populate_sources(auth_plan=false, include_mocks=true)
   end
 
   def current_plan_fixture
+    plan_name = 'current_plan'
```
Comment (MalloZup) [inline spec/support/helpers.rb:24]: minor nit. even if this are helpers, if they are supposed to be constant they should be called in higher case.

`PLAN_NAME` =

Comment (jesusbv) [reply spec/support/helpers.rb]: good catch

**Comment by bear454** [comment]
Sorry, superceded by #198. :/

**Pull request closed without merging**